cassiopeia/solver/graph_utilities.py

```diff
@@ -237,26 +237,26 @@ def spectral_improve_cut(G: nx.Graph, cut: List[int]) -> List[int]:
         A new partition that is a local minimum to the objective function
     """
 
-    def set_improvement_potential(u):
+    def set_improvement_potential(node: int):
```

Comment:
Please add full documentation to this helper function. This is still missing an Args section, a Returns section, a return type in the function declaration